CanvasPlusPlayground/PeopleManager.swift

```diff
@@ -20,39 +20,22 @@ class PeopleManager {
         self.users = []
     }
     
-    func fetchCurrentCoursePeople() async {
-        guard let courseID else {
-            print("People course Id is nil.")
-            return
-        }
-        
-        let enrollments = await fetchPeopleWith(courseID: courseID)
-        self.enrollments = enrollments
-
-        for enrollment in self.enrollments {
-            if let user = enrollment.user {
-                self.users.append(user)
+    func fetchPeople(with courseID: String) async {
```

Comment:
nit: this can be static, or use the previous method definition of using the courseID from the instance of `PeopleManager` itself. There's no need to pass in the `courseID`.